### PR TITLE
Add scatter plot of capacity vs curtailment to tutorial

### LINKED ISSUE #287: Add scatter plot of capacity vs curtailment to tutorial

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Add scatter plot of capacity vs curtailment of generators in an area to tutorial. Closes #287.

### What the code is doing
N/A

### Testing
Code snippet has been run in the **plot_tutorial** notebook.
You can run `tox` in the **docs** repository to see the updates

### Where to look
* added a bullet point for the new plot
* moved scatter plot of capacity vs cost curve slope to the **Price** section

### Usage Example/Visuals
This is the plot that is displayed:
capacity_vs_curtailment_solar_western_scatter

### Time estimate
5 min

### CODE REVIEW

File: docs/plot.rst [hunk 1]
```diff
@@ -213,6 +216,25 @@ Price
...
+  - plot capacity vs cost curve slope of generators in an area (`notebook
+    <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/scatter_capacity_vs_cost_curve_slope_demo.ipynb>`__)
+
+    .. code-block:: python
+
+        from powersimdata.scenario.scenario import Scenario
+        from powersimdata.utility.helpers import PrintManager
+        from postreise.plot.plot_scatter_capacity_vs_cost_curve_slope import (
+            plot_scatter_capacity_vs_cost_curve_slope
+        )
```
Comment (BainanXia) [inline docs/plot.rst:228]: Should we follow consistent flake8 import format in these code snippets?

Comment (rouille) [reply docs/plot.rst:228]: Ah, the newline between the powersimdata import and the postreise one

Comment (BainanXia) [reply docs/plot.rst:228]: Something like a blank line between import of other packages and the current one (as we did above).

Comment (rouille) [reply docs/plot.rst]: Done in 575337abf9e986ded5c68cd33e895247f67c6520. What needed to be done:
* organize statements alphabetically
* separate postreise modules and others with a blank line in between
* use 2 blank lines between imports and code

File: docs/plot.rst [hunk 2]
```diff
@@ -322,13 +361,14 @@ Generator Capacity and Generation
 
       import inspect
       import os
+      from powersimdata.design.generation.clean_capacity_scaling import (
+        load_targets_from_csv,
+      )
       from powersimdata.utility.helpers import PrintManager
 
       import postreise
```
Comment (BainanXia) [inline docs/plot.rst:369]: Out of scope of this PR, is this import redundant?

File: docs/analyze.rst [hunk 3]
```diff
@@ -138,9 +155,12 @@ Generator Capacity and Generation
 
   .. code-block:: python
 
+
```
[the remark this answers is not in the document]
Comment (rouille) [reply docs/analyze.rst]: fixed

**Review by BainanXia — APPROVED** [review]
Very nice.

**Pull request merged**